<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>profilesIcanView</class>
 <widget class="QWidget" name="profilesIcanView">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>771</width>
    <height>521</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QFrame" name="frame_2">
   <property name="geometry">
    <rect>
     <x>1</x>
     <y>441</y>
     <width>771</width>
     <height>81</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QFrame
{
background:rgb(81,119,128);
}</string>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
   <widget class="QPushButton" name="pushButton_14">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>20</y>
      <width>113</width>
      <height>32</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">
QPushButton
{
background:rgb(81,119,128);
border-radius:60px;
color:black;
}
QPushButton:hover
{
background:argb(214,182,151);
border-radius:60px;
color:white;
}</string>
    </property>
    <property name="text">
     <string>Home Page</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_15">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>20</y>
      <width>113</width>
      <height>32</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">
QPushButton
{
background:rgb(81,119,128);
border-radius:60px;
color:black;
}
QPushButton:hover
{
background:argb(214,182,151);
border-radius:60px;
color:white;
}</string>
    </property>
    <property name="text">
     <string>My Profile</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_16">
    <property name="geometry">
     <rect>
      <x>310</x>
      <y>21</y>
      <width>141</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">
QPushButton
{
background:rgb(81,119,128);
border-radius:60px;
color:black;
}
QPushButton:hover
{
background:argb(214,182,151);
border-radius:60px;
color:white;
}</string>
    </property>
    <property name="text">
     <string>Pictures I can view</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_17">
    <property name="geometry">
     <rect>
      <x>640</x>
      <y>20</y>
      <width>113</width>
      <height>32</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">
QPushButton
{
background:rgb(81,119,128);
border-radius:60px;
color:black;
}
QPushButton:hover
{
background:argb(214,182,151);
border-radius:60px;
color:white;
}</string>
    </property>
    <property name="text">
     <string>Requests</string>
    </property>
   </widget>
  </widget>
  <widget class="QFrame" name="frame">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>771</width>
     <height>61</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QFrame
{
background:rgb(81,119,128);
}</string>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>10</y>
      <width>251</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QLabel
{
font-size:30px;
}</string>
    </property>
    <property name="text">
     <string>Profiles I can View</string>
    </property>
   </widget>
   <widget class="QPushButton" name="Signup">
    <property name="geometry">
     <rect>
      <x>560</x>
      <y>10</y>
      <width>191</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton
{
background:rgb(214,182,151);
border-radius:60px;
color:white;
}
QPushButton:hover
{
background:argb(214,182,151);
border-radius:60px;
color:white;
}</string>
    </property>
    <property name="text">
     <string>Signout</string>
    </property>
   </widget>
  </widget>
  <widget class="QPushButton" name="pushButton_18">
   <property name="geometry">
    <rect>
     <x>470</x>
     <y>460</y>
     <width>141</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">
QPushButton
{
background:rgb(81,119,128);
border-radius:60px;
color:black;
}
QPushButton:hover
{
background:argb(214,182,151);
border-radius:60px;
color:white;
}</string>
   </property>
   <property name="text">
    <string>Profile I can view</string>
   </property>
  </widget>
  <widget class="QFrame" name="frame_3">
   <property name="geometry">
    <rect>
     <x>-1</x>
     <y>59</y>
     <width>781</width>
     <height>381</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QFrame
{
background:#333;
}</string>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>38</x>
      <y>20</y>
      <width>141</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QLabel
{
color:white;
font-size:24px;
}</string>
    </property>
    <property name="text">
     <string>@ayashaker</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>80</y>
      <width>191</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QLabel
{
color:white;
font-size:24px;
}</string>
    </property>
    <property name="text">
     <string>@hannahmontana</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>140</y>
      <width>141</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QLabel
{
color:white;
font-size:24px;
}</string>
    </property>
    <property name="text">
     <string>@taylorswift13</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>210</y>
      <width>141</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QLabel
{
color:white;
font-size:24px;
}</string>
    </property>
    <property name="text">
     <string>@justinbieber</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_6">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>280</y>
      <width>141</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QLabel
{
color:white;
font-size:24px;
}</string>
    </property>
    <property name="text">
     <string>@Barney</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_7">
    <property name="geometry">
     <rect>
      <x>400</x>
      <y>20</y>
      <width>141</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QLabel
{
color:white;
font-size:24px;
}</string>
    </property>
    <property name="text">
     <string>@ayashaker</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_8">
    <property name="geometry">
     <rect>
      <x>400</x>
      <y>80</y>
      <width>141</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QLabel
{
color:white;
font-size:24px;
}</string>
    </property>
    <property name="text">
     <string>@ayashaker</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_9">
    <property name="geometry">
     <rect>
      <x>400</x>
      <y>140</y>
      <width>141</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QLabel
{
color:white;
font-size:24px;
}</string>
    </property>
    <property name="text">
     <string>@ayashaker</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_10">
    <property name="geometry">
     <rect>
      <x>400</x>
      <y>210</y>
      <width>141</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QLabel
{
color:white;
font-size:24px;
}</string>
    </property>
    <property name="text">
     <string>@ayashaker</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_11">
    <property name="geometry">
     <rect>
      <x>400</x>
      <y>280</y>
      <width>141</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QLabel
{
color:white;
font-size:24px;
}</string>
    </property>
    <property name="text">
     <string>@ayashaker</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>30</y>
      <width>113</width>
      <height>32</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton
{
background:green;
border-radius:60px;
color:white;
}
QPushButton:hover
{
background:argb(214,182,151);
border-radius:60px;
color:white;
}</string>
    </property>
    <property name="text">
     <string>View</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_2">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>90</y>
      <width>113</width>
      <height>32</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton
{
background:green;
border-radius:60px;
color:white;
}
QPushButton:hover
{
background:argb(214,182,151);
border-radius:60px;
color:white;
}</string>
    </property>
    <property name="text">
     <string>View</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_3">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>140</y>
      <width>113</width>
      <height>32</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton
{
background:green;
border-radius:60px;
color:white;
}
QPushButton:hover
{
background:argb(214,182,151);
border-radius:60px;
color:white;
}</string>
    </property>
    <property name="text">
     <string>View</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_4">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>210</y>
      <width>113</width>
      <height>32</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton
{
background:green;
border-radius:60px;
color:white;
}
QPushButton:hover
{
background:argb(214,182,151);
border-radius:60px;
color:white;
}</string>
    </property>
    <property name="text">
     <string>View</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_5">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>290</y>
      <width>113</width>
      <height>32</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton
{
background:green;
border-radius:60px;
color:white;
}
QPushButton:hover
{
background:argb(214,182,151);
border-radius:60px;
color:white;
}</string>
    </property>
    <property name="text">
     <string>View</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_6">
    <property name="geometry">
     <rect>
      <x>590</x>
      <y>30</y>
      <width>113</width>
      <height>32</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton
{
background:green;
border-radius:60px;
color:white;
}
QPushButton:hover
{
background:argb(214,182,151);
border-radius:60px;
color:white;
}</string>
    </property>
    <property name="text">
     <string>View</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_7">
    <property name="geometry">
     <rect>
      <x>590</x>
      <y>80</y>
      <width>113</width>
      <height>32</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton
{
background:green;
border-radius:60px;
color:white;
}
QPushButton:hover
{
background:argb(214,182,151);
border-radius:60px;
color:white;
}</string>
    </property>
    <property name="text">
     <string>View</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_8">
    <property name="geometry">
     <rect>
      <x>590</x>
      <y>140</y>
      <width>113</width>
      <height>32</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton
{
background:green;
border-radius:60px;
color:white;
}
QPushButton:hover
{
background:argb(214,182,151);
border-radius:60px;
color:white;
}</string>
    </property>
    <property name="text">
     <string>View</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_9">
    <property name="geometry">
     <rect>
      <x>590</x>
      <y>210</y>
      <width>113</width>
      <height>32</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton
{
background:green;
border-radius:60px;
color:white;
}
QPushButton:hover
{
background:argb(214,182,151);
border-radius:60px;
color:white;
}</string>
    </property>
    <property name="text">
     <string>View</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_10">
    <property name="geometry">
     <rect>
      <x>590</x>
      <y>290</y>
      <width>113</width>
      <height>32</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton
{
background:green;
border-radius:60px;
color:white;
}
QPushButton:hover
{
background:argb(214,182,151);
border-radius:60px;
color:white;
}</string>
    </property>
    <property name="text">
     <string>View</string>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
